aten/src/ATen/StorageImpl.h

```diff
@@ -108,9 +109,9 @@ struct AT_API StorageImpl : public c10::raw_intrusive_ptr_target<StorageImpl> {
   at::Allocator* allocator() {
     return allocator_;
   };
-  at::ScalarType scalar_type() const {
-    return scalar_type_;
-  };
+  const DataType& dtype() const {
```

Comment:
no reference here either.

Reply:
Will change. It got it from [caffe2](https://github.com/pytorch/pytorch/blob/master/caffe2/core/storage.h#L54).

Reply:
Would be good to fix up Caffe2, in that case.

Reply:
Fixed in master at this point after the merge